src/requests.rs

```diff
@@ -619,3 +640,45 @@ impl TryFrom<OriginalCrossSigningBootstrapRequests> for CrossSigningBootstrapReq
         })
     }
 }
+
+pub mod test {
+    #![allow(missing_docs)]
+    // create data for tests/requests.test.js
+    use std::collections::BTreeMap;
+
+    use matrix_sdk_common::ruma::{
+        api::client::keys::{
+            claim_keys::v3::Request as OriginalKeysClaimRequest,
+            upload_keys::v3::Request as OriginalKeysUploadRequest,
+        },
+        device_id, user_id, DeviceKeyAlgorithm,
+    };
+    use matrix_sdk_crypto::requests::KeysQueryRequest as OriginalKeysQueryRequest;
+    use wasm_bindgen::prelude::wasm_bindgen;
+
+    use super::{KeysClaimRequest, KeysQueryRequest, KeysUploadRequest};
+
+    #[wasm_bindgen(js_name = "_test_make_keys_claim_request")]
+    pub fn make_keys_claim_request() -> KeysClaimRequest {
+        let request = OriginalKeysClaimRequest::new(BTreeMap::from([(
+            user_id!("@alice:localhost").to_owned(),
+            BTreeMap::from([(
+                device_id!("ABCDEFG").to_owned(),
+                DeviceKeyAlgorithm::SignedCurve25519,
+            )]),
+        )]));
+        KeysClaimRequest::try_from(("ID".to_string(), &request)).unwrap()
+    }
+
+    #[wasm_bindgen(js_name = "_test_make_keys_query_request")]
```

Comment:
It's too bad that it's going to be part of the public API :-/. Can't we use https://docs.rs/wasm-bindgen-test/ to address that?

Reply:
done